src/settings/OrderTemplates/OrderTemplateView/OrderTemplateView.js

```diff
@@ -159,21 +159,20 @@ class OrderTemplateView extends Component {
       contributorNameTypes,
     } = this.props;
     const { sections, showConfirmDelete } = this.state;
-    const orderTemplate = get(template, 'orderTemplate', {});
```

Comment:
was it necessary to change `orderTemplate` name to `template`? Isn't it easier to change one line instead - `template: orderTemplate,`?

Reply:
changed